ksql-engine/src/test/java/io/confluent/ksql/function/udaf/array/CollectSetUdafTest.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ **/
+
+package io.confluent.ksql.function.udaf.array;
+
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.hasSize;
+import static org.junit.Assert.assertThat;
+import io.confluent.ksql.function.udaf.Udaf;
+import java.util.List;
+import org.junit.Test;
+
+public class CollectSetUdafTest {
+
+  @Test
+  public void shouldCollectDistinctInts() {
+    final Udaf<Integer, List<Integer>> udaf = CollectSetUdaf.createCollectSetInt();
+    final Integer[] values = new Integer[] {3, 4, 5, 3};
+    List<Integer> runningList = udaf.initialize();
+    for (final Integer i : values) {
+      runningList = udaf.aggregate(i, runningList);
+    }
+    assertThat(runningList, hasSize(3));
```

Comment:
nit: line is redundant. The `contains` check on the next line will fail if the size is wrong.

Reply:
removed all these